Polyfills/Window/readme.md

```diff
@@ -0,0 +1,6 @@
+# Window
+Implements parts of the `window` object. Currently supports:
+* `setTimeout()` but not `clearTimeout()`, `setInterval()`, or `clearInterval()`
+* `addEventListener()` and `removeEventListener()`
```

Comment:
This is false. These functions are stubs.

Reply:
good catch! thanks